assets/templates/partials/banners/search-beta.tmpl

```diff
@@ -3,7 +3,7 @@
         <div class="col col--md-40 col--lg-44">
             <span class="banner__text-icon {{ if eq .Type "search" }} banner__text-icon--double-line {{ end }}">{{ localise "Beta" .Language 1 }}</span>
             <div class="banner__body">
-                <p>{{ localise "BetaCMDFeedback" .Language 1 | safeHTML }} </p>
+                <p>{{ localise "BetaSearchFeedback" .Language 1 | safeHTML }} </p>
                 {{ if eq .Type "search" }} 
                     <form action="/search" method="get">
                         <input type="hidden" id="q" name="q" value="{{ .Data.Query }}">
```

Comment:
I realise this is scope creep so feel free to ignore, but on line 11 below the text `Return to the previous version` should be using a dictionary code.